docs/platform-integration/storage/secure-storage.md

```diff
@@ -0,0 +1,151 @@
+ms.date: 08/27/2021
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.Storage", "SecureStorage"]
+#acrolinx score 95
+---
+
+# Secure storage
+
+This article describes how you can use the .NET Multi-platform App UI (.NET MAUI) `ISecureStorage` interface. This interface helps securely store simple key/value pairs. The `ISecureStorage` interface is exposed through the `SecureStorage.Default` property.
+
+[!INCLUDE [docs under construction](~/includes/preview-note.md)]
+
+The `SecureStorage` and `ISecureStorage` types are available in the `Microsoft.Maui.Storage` namespace.
+
+## Get started
+
+To access the **SecureStorage** functionality, the following platform-specific setup is required:
+
+<!-- markdownlint-disable MD025 -->
+# [Android](#tab/android)
+
+[Auto Backup for Apps](https://developer.android.com/guide/topics/data/autobackup) is a feature of Android 6.0 (API level 23) and later that backs up user's app data (shared preferences, files in the app's internal storage, and other specific files). Data is restored when an app is reinstalled or installed on a new device. This can affect `SecureStorage`, which utilizes share preferences that are backed up and can't be decrypted when the restore occurs. .NET MAUI automatically handles this case by removing the key so it can be reset. Alternatively, you can disable Auto Backup.
+
+### Enable or disable backup
+
+You can choose to disable Auto Backup for your entire application by setting `android:allowBackup` to false in the _AndroidManifest.xml_ file. This approach is only recommended if you plan on restoring data in another way.
+
+```xml
+<manifest ... >
+    ...
+    <application android:allowBackup="false" ... >
+        ...
+    </application>
+</manifest>
+```
+
+### Selective backup
+
+Auto Backup can be configured to disable specific content from backing up. You can create a custom rule set to exclude `SecureStore` items from being backed up.
+
+01. Set the `android:fullBackupContent` attribute in your _AndroidManifest.xml_:
+
+    ```xml
+    <application ...
+        android:fullBackupContent="@xml/auto_backup_rules">
+    </application>
+    ```
+
+01. Create a new XML file named _auto_backup_rules.xml_ in the _Resources/xml_ directory with the build action of **AndroidResource**. Set the following content that includes all shared preferences except for `SecureStorage`:
+
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <full-backup-content>
+        <include domain="sharedpref" path="."/>
+        <exclude domain="sharedpref" path="${applicationId}.mauiessentials.xml"/>
+    </full-backup-content>
+    ```
+
+# [iOS](#tab/ios)
+
+When developing on the **iOS simulator**, enable the **Keychain** entitlement and add a keychain access group for the application's bundle identifier.
+
+Open the _Entitlements.plist_ in the project and find the **Keychain** entitlement and enable it. This will automatically add the application's identifier as a group.
```

Comment:
This one will be interesting, because the project template doesn't add an Entitlements file. But if the user adds their own (no template to do so), it will open in the entitlements editor, which I believe isn't up to date.

I'd leave this in. Just saying.

Reply:
Maybe just link to the (very basic) entitlements doc? https://docs.microsoft.com/en-gb/dotnet/maui/ios/deployment/entitlements